src/components/Presentation/index.tsx

```diff
@@ -63,14 +96,31 @@ export const Presentation = () => {
           <StyledButton variant="contained" onClick={openPaperHandler}>
             {t('presentation.buttons.whitepaper')}
           </StyledButton>
-          <StyledButton
+          <DiscordButton
             variant="contained"
             color="secondary"
             onClick={openDiscordInvite}
             endIcon={<IconDiscord color="white" />}
           >
-            {t('presentation.buttons.community')}
-          </StyledButton>
+            {useIsMobile() ? '' : (t('presentation.buttons.community'))}
+          </DiscordButton>
+          <VideoButton
+            variant="contained"
+            color="info"
+            onClick={handleOpen}
+            endIcon={<PlayCircleOutlineRoundedIcon color="white" />}
+          />
+
+          <Modal
+            open={open}
+            onClose={handleClose}
+            aria-labelledby="modal-modal-title"
+            aria-describedby="modal-modal-description"
```

Comment:
this aria texts should be something related to the video I guess, like: "TheBadge video"